ietf/utils/db.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The IETF Trust 2021, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+# Taken from/inspired by
+# https://stackoverflow.com/questions/55147169/django-admin-jsonfield-default-empty-dict-wont-save-in-admin
+# 
+# JSONField should recognize {}, (), and [] as valid, non-empty JSON
+# values.  However, the base Field class excludes them
+import jsonfield
+from django.forms import fields
+
+from ietf.utils.fields import IETFJSONField
+
+
+class IETFJSONField(jsonfield.JSONField):
+    form_class = IETFJSONField
+    
+    def __init__(self, *args, empty_values=fields.CharField.empty_values, allowed_empty_values=[], **kwargs):
```

Comment:
Can you grab `ietf.utils.fields.IETFJSONField` here? That would guarantee that the two classes have the same idea about what the default `empty_values` is.

In that class, would it make sense to use `jsonfield.fields.forms.JSONField.empty_values` instead of jumping up to `CharField`?

Reply:
> In that class, would it make sense to use jsonfield.fields.forms.JSONField.empty_values instead of jumping up to CharField?

Yeah, that makes sense - this was probably a result of doing the work first on a clone of the upstream repository, and then copying it into here.  In that context, I think using CharField makes sense, but less so here.

Reply:
> Can you grab ietf.utils.fields.IETFJSONField here? That would guarantee that the two classes have the same idea about what the default empty_values is.

I'm not sure what you're suggesting here.  Could you elaborate?

Reply:
> > Can you grab ietf.utils.fields.IETFJSONField here? That would guarantee that the two classes have the same idea about what the default empty_values is.
> 
> I'm not sure what you're suggesting here. Could you elaborate?

I think I meant to say `ietf.utils.fields.IETFJSONField.empty_values` in the first line but left out the `empty_values` part. It looks to me like that's what you have done.